packages/rocketchat-file-upload/client/lib/FileUploadAmazonS3.js

```diff
@@ -8,7 +8,7 @@ FileUpload.AmazonS3 = class FileUploadAmazonS3 extends FileUploadBase {
 
 	start() {
 		this.uploader.send(this.file, (error, downloadUrl) => {
-			var file, item, uploading;
+			let item, uploading;
```

Comment:
We can remove this line and use `const` in the code below. Probably this file was converted from coffee using the compiler, that is why this line is here :(